### PR TITLE
Fix out of date documentation

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

File: scripts/docker-init-database [hunk 1]
```diff
@@ -1,9 +0,0 @@
-#!/bin/bash -e
-
-export PGPASSWORD=${DB_1_ENV_PGSQL_ROLE_1_PASSWORD}
-
-psql -h ${DB_1_PORT_5432_TCP_ADDR} -U nodewatcher -c "SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE datname = 'nodewatcher';" 2>/dev/null || true
-dropdb -h ${DB_1_PORT_5432_TCP_ADDR} -U nodewatcher --if-exists nodewatcher
-createdb -h ${DB_1_PORT_5432_TCP_ADDR} -U nodewatcher -E UNICODE -O nodewatcher nodewatcher
-psql -h ${DB_1_PORT_5432_TCP_ADDR} -U nodewatcher -d nodewatcher -c "CREATE EXTENSION postgis;"
-psql -h ${DB_1_PORT_5432_TCP_ADDR} -U nodewatcher -d nodewatcher -c "CREATE EXTENSION postgis_topology;"
```
Comment (mitar) [inline scripts/docker-init-database:9]: The script [here](https://github.com/tozd/docker-postgresql/blob/master/etc/service/postgresql/run) does not create `postgis_topology`. Is this OK?

**Review by mitar — APPROVED** [review]
I think it is OK. Check that one my comment.

**Comment by mitar** [comment]
BTW, with new Docker on Mac, the instructions now hold even for that. Because it maps `/tmp` into the virtual machine running Docker.

File: scripts/docker-init-database (hunk 1 above)
Comment (kostko) [reply scripts/docker-init-database:9]: I've been using it this way and haven't run into any problems so far, so I think it is ok.

**Pull request merged**